### PR TITLE
test commit for github branch/issues/pull request testing.

### PR DESCRIPTION
**Pull request by sudreg**
for testing of github pull request..

### CODE REVIEW

File: Code/Websites/DanpheEMR/Controllers/Appointment/AppointmentController.cs [hunk 1]
```diff
@@ -38,6 +38,8 @@ namespace DanpheEMR.Controllers
 
     public class AppointmentController : CommonController
     {
+        //comment added for testing of git branches: sud-6Feb2023
```
Comment (bikeshbist123) [inline Code/Websites/DanpheEMR/Controllers/Appointment/AppointmentController.cs]: this line is not required so please remove this

**Review by bikeshbist123 — CHANGES_REQUESTED** [review]
there are some pending issues please complete those first

File: Code/Websites/DanpheEMR/Controllers/Appointment/AppointmentController.cs (hunk 1 above)
Comment (sudreg) [reply Code/Websites/DanpheEMR/Controllers/Appointment/AppointmentController.cs]: Removed the mentioned line.

**Review by bikeshbist123 — APPROVED** [review]
okey review is done proceed for merging

**Pull request merged**